data/map planets.txt

```diff
@@ -123,8 +123,8 @@ planet Nareltok
 planet Ngakego
 	attributes factory naltok urban
 	landscape land/city7
-	description "Ngakego is by far the largest shipyard in Telashki space, as well as being one of the more populous planets. The warships assembled here mostly go on to join the defenses on Kolxacha, while many of the freighters built here are back within days carrying fresh components."
-	spaceport `The spaceport here, like many Naltok buildings, is filled with about a foot of lukewarm water. For the Naltok this is essential, in order to prevent their skin from drying out in the air, but for humans and any other visitors this is inconvenient and highly unpleasant. Unlike many other Naltok buildings however, this building has "Visitor paths" which are almost entirely dry, allowing humans and alphas to traverse most of the spaceport in comfort.`
+	description `Ngakego is by far the largest shipyard in Telashki space, as well as being one of the more populous planets. The warships assembled here mostly go on to join the defenses on Kolxacha, while many of the freighters built here are back within days carrying fresh components.`
+	spaceport `The spaceport here, like many Naltok buildings, is filled with about a foot of lukewarm water. For the Naltok this is essential, in order to prevent their skin from drying out in the air, but for humans and any other visitors this is inconvenient and highly unpleasant. Unlike many other Naltok buildings however, this building has "Visitor paths" which are almost entirely dry, allowing humans and Alphas to traverse most of the spaceport in comfort.`
```

Comment:
Should `humans` be capitalised as well? It seems odd to have humans lowercase and alphas capitalised, particularly when they're right next to each other in the sentence.

Reply:
Would it not follow on from that that the names of other species would be lower-case too?